src/firefly/java/edu/caltech/ipac/visualize/plot/CropFile.java

```diff
@@ -140,8 +140,8 @@ static public void crop_extensions(String in_filename, String out_filename,
                 ImageHDU h = (ImageHDU) hdu;
                 Header old_header = h.getHeader();
                 int naxis = old_header.getIntValue("NAXIS");
-                if (naxis == 0) {
-            /* it's a null image - probably the primary image */
+                if (naxis < 2 ){ //Image has to have naxis=2 or 3
```

Comment:
This is good.  I am glad you found that our image check should include checks for at least two axis.  I think this check is now in 3 places in the last two pull request.  This is good and the code is more robust